console/webvirtmgr-console

```diff
@@ -141,13 +144,13 @@ class CompatibilityMixIn(object):
          console_socket) = get_connection_infos(token)
```

Comment:
I made problably a mistake (wrong copy/paste). You can remove this last out of the scope of the patch.